packages/ng/form-field/form-field.component.ts

```diff
@@ -0,0 +1,122 @@
+import { AfterViewInit, booleanAttribute, Component, ContentChild, HostBinding, inject, Input, OnChanges } from '@angular/core';
+import { NgClass, NgIf, NgSwitch, NgSwitchCase, NgTemplateOutlet } from '@angular/common';
+import { InputDirective } from './input.directive';
+import { FormFieldSize } from './form-field-size';
+import { BehaviorSubject } from 'rxjs';
+import { InlineMessageComponent } from '../inline-message/inline-message.component';
+import { InlineMessageState } from '../inline-message/inline-message-state';
+import { SafeHtml } from '@angular/platform-browser';
+import { LuTooltipModule } from '../tooltip/tooltip.module';
+import { ReactiveFormsModule } from '@angular/forms';
+
+let nextId = 0;
+
+@Component({
+	selector: 'lu-form-field',
+	standalone: true,
+	imports: [NgIf, NgSwitch, NgSwitchCase, NgTemplateOutlet, InlineMessageComponent, LuTooltipModule, ReactiveFormsModule],
+	templateUrl: './form-field.component.html',
+	styleUrls: ['./form-field.component.scss'],
+	hostDirectives: [NgClass],
+})
+export class FormFieldComponent implements OnChanges, AfterViewInit {
+	#ngClass = inject(NgClass);
+
+	@HostBinding('class')
+	clazz = 'form-field';
+
+	@Input({
+		required: true,
+	})
+	label: string;
+
+	@Input({
+		transform: booleanAttribute,
+	})
+	hiddenLabel = false;
+
+	@Input()
+	tooltip: string | SafeHtml;
+
+	@Input({
+		transform: booleanAttribute,
+	})
+	required = false;
+
+	@Input({
+		transform: booleanAttribute,
+	})
+	invalid = false;
+
+	@Input()
+	inlineMessage: string;
+
+	/**
+	 * State of the inline message, will be ignored if form state is invalid
+	 */
+	@Input()
+	inlineMessageState: InlineMessageState;
+
+	@Input()
+	size: FormFieldSize = 'M';
+
+	@Input()
+	layout: 'default' | 'checkbox' = 'default';
+
+	@ContentChild(InputDirective)
+	input: InputDirective;
+
+	id: string;
+
+	ready$ = new BehaviorSubject<boolean>(false);
```

Comment:
We could complete this subject in ngOnDestroy to make sure that no memory leak occurs